libs/scripts/delayed.js

```diff
@@ -63,7 +63,7 @@ const loadDelayed = ([
     if (getMetadata('interlinks') === 'on') {
       const { locale } = getConfig();
       const path = `${locale.contentRoot}/keywords.json`;
-      const language = locale.lang ?? locale.ietf?.split('-')[0];
```

Comment:
@Ruchika4 I think you are the only other person to use this feature. Is this ok?

Reply:
Yes this looks good.